<launch>

	<node name="lane_follow" pkg="lane_follow_iarrc" type="lane_follow" output="screen">
        <rosparam param="angular_speed_multiplier">1</rosparam>
        <rosparam param="linear_speed_multiplier">1</rosparam>
        <rosparam param="angular_vel_cap">1</rosparam>
        <rosparam param="linear_vel_cap">1</rosparam>

        <!-- Input topic -->
        <remap from="vision/complete_filtered_image" to="vision/ipm_output_image"/>
	</node>

</launch>